### DISCUSSION TITLE
Move mbedtls to 2.28 LTS version

### DISCUSSION THREAD

**Discussion by gabekassel** [openthread/openthread#7527] [Ideas]
mbedtls 2.28 is an LTS version with security fixes from the current 2.25 OT is using. Any compelling reason not to move to 2.28? 

https://github.com/ARMmbed/mbedtls/blob/development/BRANCHES.md

@jwhui @LuDuda

**Comment by jwhui** [comment]
I don't have any reason not to move from 2.25 to 2.28.

I just submitted https://github.com/openthread/openthread/pull/7539, please review.

**Reply by jwhui** [reply]
#7539 has been merged.
